gp2gp-translator/build.gradle

```diff
@@ -27,6 +27,15 @@ dependencies {
     implementation project(':common')
     implementation project(':db-connector')
     implementation project(':schema')
+    implementation 'org.springframework.boot:spring-boot-starter-actuator'
```

Comment:
All of those libs are inside the new common module, so they're not needed here anymore. Could you just update the version of `org.springframework:spring-jms:5.3.13` inside the common module please? (I can see older version there).